Vector.sc

```diff
@@ -1,92 +1,106 @@
-// �2009 Miguel Negr�o, Fredrik Olofsson
+// 2009 Miguel Negro, Fredrik Olofsson
```

Comment:
Please correct to 

    Copyright 2009 Miguel Negrão, Fredrik Olofsson

Reply:
No problem.